### PR TITLE
Wallet: Remove the requirement to be authenticated, add fallback when no location is set

### PR DESCRIPTION
**Pull request by dsluijk**
People are running into a generic error when trying to add a ticket to their {Apple, Google} Wallet. Since the ticket key is sufficiently random, we can just remove the requirement to be authenticated.

Secondly for the Google Wallet we get an error when no location is set for a ticket. This also adds a fallback location (Unknown), so it still proceeds.

### CODE REVIEW

File: .devcontainer/devcontainer.json
Comment (JoepdeJong) [inline .devcontainer/devcontainer.json:1]: Is this on purpose?

**Review by JoepdeJong — CHANGES_REQUESTED** [review]
LGTM. just check the devcontainer file, not sure if you committed that on purpose.

File: .devcontainer/devcontainer.json
Comment (dsluijk) [reply .devcontainer/devcontainer.json:1]: Yes, I ran into an issue with the new version of the Java version, so I set a more specific version. The rest is just autoformatting

**Review by JoepdeJong — APPROVED** [review]

**Comment by JoepdeJong** [comment]
@dsluijk i just noticed, as authenticationService is not required anymore, it can be removed from the initializer. can you look at this?

**Comment by dsluijk** [comment]
I believe this is still required for other subclasses of the WebshopController superclass. So I don't think we can do that.

**Pull request merged**